### DISCUSSION TITLE
2024 Cohort 여러분들을 진심으로 환영합니다 🤩

### DISCUSSION THREAD

**Discussion by yeniful** [DeveloperAcademy-POSTECH/.github#13] [Announcements]
(본 미션은 2024 Cohort를 대상으로 한 미션입니다.)

## Tech Forum Mission
테크 포럼의 공지사항을 찾아내셨군요! 대단한걸요? 😎🙌

우리가 수행해야하는 테크 포럼 미션은 에어테이블에 깃허브 유저네임을 제출하는 것입니다!
우리는 유저네임을 이용해서 계정을 찾아가거나, 호출하고싶은 사람들을 태그할 수 있답니다. @M1zz 이렇게요!
 
4월 15일까지 하단의 에어테이블 링크를 통해 여러분들의 깃허브 유저네임을 알려주세요 👀
또한 폼 제출 후 유저네임을 바꾸게 된다면 Teams를 통해서 꼭 알려주세요!

https://airtable.com/app22MrSLBbhZB9jV/pagZ4e0z2tHlTQncm/form

**Comment by Dirini** [comment]
안녕하세요 :)

**Comment by penum1227** [comment]
우왕

**Comment by jangjia01234** [comment]
우와 포럼! 깃허브 친구해요 여러분 🙌🍀

**Comment by Monfi98** [comment]
신기방구

**Comment by insub4067** [comment]
저도 껴주나요
